interface/modules/custom_modules/oe-module-claimrev-connect/src/Eligibility.php

```diff
@@ -0,0 +1,237 @@
+<?php 
```

Comment:
This is not a class (ie. is a script), so belongs in templates dir

Reply:
moved